pkg/config/corscfg/options.go

```diff
@@ -3,7 +3,16 @@
 
 package corscfg
 
+import "go.opentelemetry.io/collector/config/confighttp"
+
 type Options struct {
 	AllowedOrigins []string
 	AllowedHeaders []string
 }
+
+func (o *Options) ToOTELCorsConfig() *confighttp.CORSConfig {
```

Comment:
these look to be the exact same so can we just remove this type altogether and only use the one from OTEL?

Reply:
double checking on this - is there a reason we can't remove this type?